frontend/src/lib/components/TZLabel/index.tsx

```diff
@@ -85,17 +86,21 @@ const TZLabelPopoverContent = React.memo(function TZLabelPopoverContent({
 })
 
 /** Return a simple label component with timezone conversion UI. */
-function TZLabelRaw({
-    time,
-    showSeconds,
-    formatDate,
-    formatTime,
-    showPopover = true,
-    noStyles = false,
-    className,
-    children,
-    ...dropdownProps
-}: TZLabelProps): JSX.Element {
+
+const TZLabelRaw = forwardRef<HTMLElement, TZLabelProps>(function TZLabelRaw(
```

Comment:
Ended up not needing this but I think it's worth having. Allows a tooltip to be placed on the `TZLabel` component